CMakeLists.txt

```diff
@@ -42,9 +42,10 @@ project (Cocos2d-X)
 set(COCOS2D_X_VERSION 3.13)
 
 # define some variables
-
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${PROJECT_SOURCE_DIR}/cmake/Modules/")
 set(COCOS_EXTERNAL_DIR ${CMAKE_CURRENT_SOURCE_DIR}/external)
+set(COCOS_STATIC_LIBS_PATH ${PROJECT_SOURCE_DIR}/prebuilt)
```

Comment:
COCOS_STATIC_LIBS_PATH -> COCOS_PREBUILT_LIBS_PATH

It makes it more meaningful.